### ISSUE TITLE
ResNet loss doesn't change

### ISSUE THREAD

**Issue by TKassis** [uber/ludwig#402]
**Describe the bug**
When running the ResNet encoder, and no matter what hyperparameters I set, I keep getting extremely high losses, train accuracy of 0.4949, vali 0.5261 and test 0.5050 (always these values). And loss does not increase at all even after 15 epochs. My test dataset is the dogs vs cats Kaggle dataset which has around 12k of each.

Please provide code, yaml definition file and a sample of data in order to entirely reproduce the issue.
Issues that are not reproducible will be ignored.

The model_definition file is as follows. I've tried several different learning rates, batch sizes, dropouts and resent sizes (34, 50)

```
input_features:
    -
        name: image_path
        type: image
        encoder: resnet
        resnet_size: 50
        dropout: true
        preprocessing:
          resize_method: interpolate
          height: 224
          width: 224

output_features:
    -
        name: label
        type: category

training:
    batch_size: 16
    learning_rate: 0.01
    dropout_rate: 0.5
    epochs: 25
```

**Expected behavior**
I expect to see a decrease in loss with each epoch.

**Screenshots**
I attached all the screenshots
[Screenshots and files.zip](https://github.com/uber/ludwig/files/3337131/Screenshots.and.files.zip)

**Environment (please complete the following information):**
 - OS: Windows
 - Version: 10
- Python version: 3.6
- Ludwig version: 0.1.2

**Additional context**

**Comment by w4nderlust** [comment]
It's really likely your learning rate is too high, you should use something around 0.0001 . Also, make sure you are using the latest code from master as there have been some fixes specifically related to resnets.
Closing as this is not an issue, but just an hyperaprameter misconfiguration. Feel free to keep posting if you keep encountering issues.

**Comment by TKassis** [comment]
I'm still having the same issue. I've been looking at the hdf5 file created and there are a lot of negative values. Shouldn't these be all normalized positive values? 

![Annotation 2019-06-30 081818](https://user-images.githubusercontent.com/9168908/60396434-a4033880-9b0f-11e9-95e7-463771cfda42.png)

[experiment_run_1.zip](https://github.com/uber/ludwig/files/3342859/experiment_run_1.zip)
